microsoft-edge/webview2/release-notes/index.md

```diff
@@ -120,6 +123,299 @@ description
+
+* [ICoreWebView2ScreenCaptureStartingEventArgs](/microsoft-edge/webview2/reference/win32/icorewebview2screencapturestartingeventargs?view=webview2-1.0.2895-prerelease&preserve-view=true)
+  * [ICoreWebView2ScreenCaptureStartingEventArgs::get_Cancel](/microsoft-edge/webview2/reference/win32/icorewebview2screencapturestartingeventargs?view=webview2-1.0.2895-prerelease&preserve-view=true#get_cancel)
+  * [ICoreWebView2ScreenCaptureStartingEventArgs::get_Handled](/microsoft-edge/webview2/reference/win32/icorewebview2screencapturestartingeventargs?view=webview2-1.0.2895-prerelease&preserve-view=true#get_handled)
+  * [ICoreWebView2ScreenCaptureStartingEventArgs::get_OriginalSourceFrameInfo](/microsoft-edge/webview2/reference/win32/icorewebview2screencapturestartingeventargs?view=webview2-1.0.2895-prerelease&preserve-
+view=true#get_originalsourceframeinfo)
+  * [ICoreWebView2ScreenCaptureStartingEventArgs::GetDeferral](/microsoft-edge/webview2/reference/win32/icorewebview2screencapturestartingeventargs?view=webview2-1.0.2895-prerelease&preserve-view=true#getdeferral)
+  * [ICoreWebView2ScreenCaptureStartingEventArgs::put_Cancel](/microsoft-edge/webview2/reference/win32/icorewebview2screencapturestartingeventargs?view=webview2-1.0.2895-prerelease&preserve-view=true#put_cancel)
+  * [ICoreWebView2ScreenCaptureStartingEventArgs::put_Handled](/microsoft-edge/webview2/reference/win32/icorewebview2screencapturestartingeventargs?view=webview2-1.0.2895-prerelease&preserve-view=true#put_handled)
+
+* [ICoreWebView2ScreenCaptureStartingEventHandler](/microsoft-edge/webview2/reference/win32/icorewebview2screencapturestartingeventhandler?view=webview2-1.0.2895-prerelease&preserve-view=true)
+
+---
+
+
+<!-- ---------- -->
+###### Configure the security warning when saving a file (`SaveFileSecurityCheckStarting` event)
+
+<!-- within Oct Prerelease SDK -->
+
+<!-- todo: was this simultaneously promoted in both SDKs?  it's in both sections (Release, Prerelease)
+promoted to Stable in Oct Release SDK
+promoted from Experimental to Stable in Oct Prerelease SDK
+-->
+
+Added a new `SaveFileSecurityCheckStarting` event.  Your app can register a handler on this event to get the file path, filename extension, and document origin URI information.  You can then apply your own rules to do actions such as the following:
+   * Allow saving the file without presenting a default security-warning UI about the file-type policy.
+   * Cancel the saving.
+   * Create your own UI to manage runtime file-type policies.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* `CoreWebView2` Class:
+    * [CoreWebView2.SaveFileSecurityCheckStarting Event](/dotnet/api/microsoft.web.webview2.core.corewebview2.savefilesecuritycheckstarting?view=webview2-dotnet-1.0.2895-prerelease&preserve-view=true)<!-- todo: getDeferral? -->
```

Comment:
add getDeferral?
@victorhuangwq

Reply:
Refer to comment below

Reply:
result 2x:
```
##### [.NET/C#](#tab/dotnetcsharp)

<!-- todo: equivalent?
.NET: CoreWebView2SaveFileSecurityCheckStartingEventArgs.GetDeferral Method
WinRT: CoreWebView2.GetDeferral Property
Win32: ICoreWebView2SaveFileSecurityCheckStartingEventArgs::GetDeferral
-->
```

Reply:
closing present comment b/c it's better; it asks Dev